libs/pcre2/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=pcre2
-PKG_VERSION:=10.23
+PKG_VERSION:=10.30
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/
-PKG_MD5SUM:=b2cd00ca7e24049040099b0a46bb3649
-PKG_HASH:=dfc79b918771f02d33968bd34a749ad7487fa1014aeb787fad29dd392b78c56e
+PKG_MD5SUM:=d3adf4b130eed854a530390f00020a65
```

Comment:
Delete the `PKG_MD5SUM`, it's causing Travis to bail:

`WARNING: Use of PKG_MD5SUM is deprecated, switch to PKG_HASH`